src/extension.ts

```diff
@@ -24,4 +24,24 @@ export async function activate(context: vscode.ExtensionContext) {
       }));
 
   await clangdContext.activate(context.globalStoragePath, outputChannel);
+
+  setInterval(function() {
+    const cpptools = vscode.extensions.getExtension('ms-vscode.cpptools');
+    if (cpptools !== undefined && cpptools.isActive) {
+      const cpptoolsConfiguration = vscode.workspace.getConfiguration('C_Cpp');
+      const cpptoolsEnabled = cpptoolsConfiguration.get('intelliSenseEngine');
+      if (cpptoolsEnabled !== 'Disabled') {
+        vscode.window
+            .showWarningMessage(
+                'You have Microsoft C++ (ms-vscode.cpptools) extension ' +
```

Comment:
We should be careful about the wording here, as many users won't have a sophisticated understanding of plugins vs features of plugins etc, but we expect them to be able to undo this change if needed.
I think we need to call out that we're only disabling IntelliSense here.

Maybe:

You have both the Microsoft C++ (cpptools) extension and clangd extension enabled.
The Microsoft IntelliSense features conflict with clangd's code completion, diagnostics etc.
[Disable IntelliSense]